TicTacToe/Program.cs

```diff
@@ -11,6 +11,7 @@ static void Main(string[] args)
             Game game = new Game();
             cli.WelcomeToGame();
             game.AddPlayers(cli);
+            game.DisplayCurrentBoard(cli);

```

Comment:
I'm not sure what your plan for this board, but it might be helpful to add numbers to this console

![Screen Shot 2020-02-24 at 5 14 42 PM](https://user-images.githubusercontent.com/29721784/75205689-48825c80-5729-11ea-86a4-344e74d527cc.png)
For example, It'd be great if I knew that typing `2` would correspond with the 2nd box. I think it will be easier for players to play the game without having to count out the boxes

Reply:
Yes, that was my plan actually! I'm still thinking about UX/UI things, but I am planning to add numbers to each space (I was also deciding on updating the initial board to contain the numbers, or simply render them when the game displays to the console)